server/server.js

```diff
@@ -125,6 +125,18 @@ module.exports = function(__dirname) {
 				value: body.content
 			}
 		}
+		if (!(body.tags.length > 1 || body.tags[0] || body.title || body.content) && modify) {
+			return new Promise(function(resolve, reject) {
```

Comment:
mongojs doesn't have promises by default, but there are a couple packages that wrap it with promises. We can add one of those, but I would definitely recommend sticking with mongo.js. It's by far the most simple and straightforward mongo driver I've used